plugins/serializers/nsca/nsca.go

```diff
@@ -0,0 +1,121 @@
+package nsca
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/influxdata/telegraf"
+)
+
+const (
+
+	// CPU load is in warning state
+	// if cpu core is 4 then warning thresold is 3.6(90%)
+	STATE_WARNING = 3.6
```

Comment:
These should be configurable I would have thought?

Also, what is the machine has 8 cores?  How can this work on anything but a machine with 4 cores?

I would have thought CPU state warnings should be a % not a value of CPU cores. Take an input of 90%, check how many cores the machine has and work out the value to check against that way.

Am I missing something?

Reply:
Yeah, we can configure.

The values  CPU state we get from input plugin is not in percentage. 

I followed below link for Nagios CPU Load setup

https://serverfault.com/questions/209566/what-warning-and-critical-values-to-use-for-check-load

Reply:
Ah ok, so it's "config-per-server" rather than one global setup.